src/ipfs/types.ts

```diff
@@ -1,4 +1,6 @@
-export type CID = string
+import type { CID as MultiformatsCID } from "multiformats/cid"
+
+export type CID = MultiformatsCID
```

Comment:
Shouldn't we just remove all of this, and use `import { CID } from "multiformats/cid"` everywhere?

Reply:
Don't know 🤷‍♂️ What if the `CID` type changes again?! 😂

Reply:
I think we're using `import { CID } from "multiformats/cid"` in some places already and we should go for consistency? If it happens to change again, find-replacing that import should do the same thing. It's just a *tiny bit* less indirections, but not a big deal to be honest.